### PR TITLE
Test using JIT as a service

### PR DESCRIPTION
**Pull request by mihaibudiu**

### CODE REVIEW

File: sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/OtherTests.java [hunk 1]
```diff
@@ -96,6 +98,38 @@ private DBSPCompiler compileDef() {
         return compiler;
     }
 
+    // Test using the JIT executor as a service.
```
Comment (lalithsuresh) [inline sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/OtherTests.java:101]: Why is this a service? It looks like the cargo command to run the JIT starts a process that reads inputs and then terminates with an exit code?

**Review by lalithsuresh — APPROVED** [review]

**Comment by mihaibudiu** [comment]
@Kixiron in a new commit I have added a "view" field to the Sink nodes, and I am using that one in the config file.
Can you please review the Rust part?

**Comment by mihaibudiu** [comment]
Funny, the java test using this feature ran, but the build still fails. Let me check what's wrong.

**Review by Kixiron — APPROVED** [review]

File: sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/OtherTests.java (hunk 1 above)
Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/OtherTests.java:102]: That process is a service.
Previously we used to link the test in the same executable.
Once this works we'll use it for all our JIT tests.

**Comment by mihaibudiu** [comment]
I think I am done with this PR. If it works I will merge - unless @lalithsuresh wants to re-review.
This will need to be fixed after #727 is fixed.

File: sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/jit/JitJsonOutputDescription.java [hunk 2]
```diff
@@ -1,37 +1,177 @@
...
+            }
+        } catch (Exception ex) {
+            throw this.parseError(lineNumber, line, ex.getMessage());
+        }
+        throw this.parseError(lineNumber, line, "Unexpected type " + type);
+    }
+
+    DBSPExpression parseLine(ObjectMapper objectMapper, long lineNumber, String line, DBSPTypeTuple elementType)
+            throws JsonProcessingException {
+        // TODO: this will have to change when the JSOn format will support weights
```
Comment (lalithsuresh) [inline sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/jit/JitJsonOutputDescription.java:145]: JSON

**Review by lalithsuresh — APPROVED** [review]

**Comment by mihaibudiu** [comment]
The last CI failure is not my fault. Something about duplicate keys in postgres.

**Pull request merged**